docker/build.sh

```diff
@@ -13,6 +13,19 @@
 set -ex
 PROJECT=rawls
 
+function build_check()
```

Comment:
I am thinking of moving this function to a common location like dsp-toolbox and calling that script from within the jenkins job, to prevent copying the same code to a bunch of locations.  will still need to change the tagging strategy below however